descriptors/wasm.go

```diff
@@ -22,6 +22,19 @@ const (
 	NetworkRegtest Network = 2
 )
 
+func ParseNetwork(network string) (Network, error) {
```

Comment:
move this to descriptors_test.go? These strings are specific to the unit test vectors only.